packages/marketplace/src/components/ui/submit-app-wizzard/form-fields.ts

```diff
@@ -0,0 +1,26 @@
+import { FormFieldInfo } from '@reapit/elements'
```

Comment:
there is an validation for submit app form. Pls try to re use it

Reply:
Data payload between submit app, submit revision are not the same now, so it can't be re-use. I built the new validation based on the standalone submit app page's form validator.